README.md

```diff
@@ -1,29 +1,90 @@
+[![CircleCI](https://circleci.com/gh/ministryofjustice/opg-metrics.svg?style=shield)](https://circleci.com/gh/ministryofjustice/opg-metrics)
+[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
+[![pre-commit](https://github.com/ministryofjustice/opg-metrics/workflows/Swagger-Documentation/badge.svg)](https://github.com/ministryofjustice/opg-metrics/workflows/Swagger-Documentation)
+
 # OPG Metrics
 
-## Integration and Mocking
+The Office of the Public Guardian metrics service: Managed by opg-org-infra &amp; Terraform
+
+Building a new metrics service for OPG.
+
+To find out more about this service, see our [about this service](ABOUT_THIS_SERVICE.md) documentation.
+
+## Prerequisites
+
+aws-vault for managing AWS access.
+
+`brew install --cask aws-vault`
```

Comment:
Is it worth linking out to https://docs.opg.service.justice.gov.uk/documentation/get_started.html#get-started